src/blocks/blocks/stripe-checkout/inspector.js

```diff
@@ -13,14 +14,71 @@ import {
 	SelectControl,
 	Spinner,
 	TextControl,
-	TextareaControl
+	TextareaControl,
+	ExternalLink
 } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
-import { ButtonToggleControl, RichTextEditor } from '../../components/index.js';
-import { useState } from '@wordpress/element';
+import { ButtonToggleControl, Notice as OtterNotice, Notice, RichTextEditor } from '../../components/index.js';
+import { setUtm } from '../../helpers/helper-functions';
+
+const ProFeatures = () => {
+	return (
+		<PanelBody
+			title={ __( 'Autoresponder', 'otter-blocks' ) }
+			initialOpen={ false }
+		>
+			<TextControl
+				label={__( 'Autoresponder Subject', 'otter-blocks' )}
+				placeholder={__(
+					'Thank you for your purchase',
+					'otter-blocks'
+				)}
+				value={ undefined }
+				onChange={ () => {}}
```

Comment:
We can just make it as disabled and won't need to have value/onChange. It's a nit.